src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -486,10 +486,10 @@ const getBoxTableData = (data = {}) => (
       .reduce(
         (tableData, stat) => (
           boxTableAllowedStats.includes(stat.toLowerCase())
-            ? tableData.concat({
-              count: data[stat],
+            ? tableData.concat((count => ({
+              count: count % 1 ? Number.parseFloat(count).toFixed(2) : count,
```

Comment:
```suggestion
              count: parseContinuousValue(count),
```

Reply:
used your function in a different manner. Thanks for pointing that one out. 👍